### PR TITLE
edit btn moved closer to username

### PR DESCRIPTION
**Pull request by ikzntsv**

### CODE REVIEW

File: resources/views/header/header.blade.php [hunk 1]
```diff
@@ -48,8 +48,8 @@
                                 </button>
                             </a>
                         </div>
-                        <div class="col-xs-4" style="margin:0; padding-top:3%">
-                                <a class="edit-user-name" href="/user/{{ Auth::user()->id }}/edit"><i class="fa fa-pencil" aria-hidden="true"></i></a>
+                        <div class="col-xs-2" style="margin-left:-20px; padding-top:3%">
```
Comment (Quicks) [inline resources/views/header/header.blade.php]: please move all styles in independent style file

Comment (ikzntsv) [reply resources/views/header/header.blade.php]: I don't know the better way to overwrite bootstrap styles rather then write style inside html)))

File: resources/views/header/header.blade.php [hunk 2]
```diff
@@ -26,32 +26,23 @@
...
-                                    </div>
-                                </button>
-                            </a>
-                        </div>
-                        <div class="col-xs-4" style="margin:0; padding-top:3%">
-                                <a class="edit-user-name" href="/user/{{ Auth::user()->id }}/edit"><i class="fa fa-pencil" aria-hidden="true"></i></a>
-                        </div>
-                    </div>
+                <div class="row pull-left">
+                    <a class="btn btn-default modal-user-button" href="/user/{{ Auth::user()->id }}/edit"><i class="fa fa-pencil" aria-hidden="true"></i></a>
```
Comment (Quicks) [inline resources/views/header/header.blade.php]: You can use route helper there

File: resources/views/header/header.blade.php [hunk 3]
```diff
@@ -26,32 +26,23 @@
...
+                    <a class="btn btn-default modal-user-button" @if(Auth::user()->isAdmin()) href="{{url('/admin')}}" @else href="{{ url('/cabinet') }}" @endif>
+                        <span class="img-user">
+                            @if(Auth::user()->avatar and File::exists(public_path('image/user/' . Auth::user()->id .'/avatar/'. Auth::user()->avatar)))
```
Comment (Quicks) [inline resources/views/header/header.blade.php]: We have function which return path to user avatar

**Pull request merged**